bundle/android/strings.xml

```diff
@@ -174,7 +174,6 @@
     <string name="gutenberg_native_inside" tools:ignore="UnusedResources">Inside</string>
     <string name="gutenberg_native_invalid_url_audio_file_not_found" tools:ignore="UnusedResources">Invalid URL. Audio file not found.</string>
     <string name="gutenberg_native_invalid_url_please_enter_a_valid_url" tools:ignore="UnusedResources">Invalid URL. Please enter a valid URL.</string>
-    <string name="gutenberg_native_line_height" tools:ignore="UnusedResources">Line Height</string>
```

Comment:
I'll keep this change since this is not released yet and it was introduced in another unrelated [PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3867).